web/js/FlexyConnector.js

```diff
@@ -26,7 +26,8 @@ function saveJSON()
    //Fetch the AppConfig
    var CycleTimeMs = document.getElementById("CycleTimeMs").value;
    var PostDuplicateTagValues = document.getElementById("PostDuplicateTagValues").value;
-   jsonObject.AppConfig = { CycleTimeMs: CycleTimeMs, PostDuplicateTagValues: PostDuplicateTagValues };
+   var CommunicationType = document.getElementById("CommunicationType").value;
+   jsonObject.AppConfig = { CycleTimeMs: CycleTimeMs, PostDuplicateTagValues: PostDuplicateTagValues, CommunicationType: CommunicationType };

```

Comment:
Options are now different and you probably need to handle the conversion from "PI Web API 2019+" -> "omf"

Reply:
The picture update got put in the web page update, and i did not think of that. I'm just going to accept both when i read the json file since that would be the easiest thing to do.

Reply:
added it in as a second option

Reply:
Oh okay, i was looking for the picture update in the readme commit. The picture is now quite large which is okay for github but if you are using another markdown viewer its pretty large on the screen. You should probably set a max width if you are using a larger resolution image.

Reply:
I prefer having images sized in advance
i'll just shrink it a bit

Reply:
it's now about as small as what github resized it to on my screen